docs/topics/extensions.md

```diff
@@ -99,6 +99,68 @@ Or they can be referenced as a preprovision extension, this will run during clou
 |---|---|---|
 |name|yes|The name of the extension. This must match the name in the extensionProfiles|
 
+## agentPoolProfiles
+
+Extensions run a script on every node in an agent pool. The extensions array in the
+agentPoolProfiles defines that an extension will be run on every node of that agent
+configuration. Extensions in agentPoolProfiles are only supported for VMAS agent pools,
+not VMSS.
+
+``` javascript

```

Comment:
```suggestion
``` json
```

Reply:
not sure why the ones above say javascript, I think they need to be changed as well 🤔